src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -95,4 +95,16 @@
      * @return True if can be interrupted, false if not
      */
     boolean canBeInterrupted();
+
+    /**
+     * Resets this task.
+     */
+    void reset();
+
+    /**
+     * Gets the mutex bits. When the | operation result of two mutex bits is 0, they can run
+     * concurrently.
+     * @return The mutex bits which categorizes this task
+     */
+    int getMutexBits();
```

Comment:
The mutex bits are an impl detail. For our API they are not required. If I can get a usecase for having them then I'll consider a child interface for a VanillaAITask of sorts but they won't go into the base interface.

@ST-DDT ....which you can get those details by calling other.getGoal().get().getOwner()...etc